lib/elastic/beanstalk/config.rb

```diff
@@ -67,15 +67,12 @@ def inactive_settings
       end
 
       def set_option(namespace, option_name, value)
-        namespace = namespace.to_sym
```

Comment:
not really, this is ensuring that the arg namespace is a sym, so that it won't fail a lookup comparison.  On read, it is deeply symbolized....wondering if this could cause a problem for someone using a programmatic #set_option but we clearly aren't failing a test on that right?